capa/features/extractors/viv/__init__.py

```diff
@@ -42,7 +38,7 @@ def add_va_int_cast(o):
     this bit of skullduggery lets use cast viv-utils objects as ints.
     the correct way of doing this is to update viv-utils (or subclass the objects here).
     """
-    setattr(o, "__int__", types.MethodType(get_va, o, type(o)))
+    setattr(o, "__int__", types.MethodType(get_va, o))
```

Comment:
we should try to maintain py2 support, if its easy, such as with `six`

Reply:
I think the code is py2 compatible. The third argument was for unbound method objects, which we are not using: https://stackoverflow.com/a/48857684/6245337

The return object is different, but in our case it works the same:
```
>>> import types
>>> class Ana:
...      def bla(self):
...          return 4
...
>>> a = Ana()
>>> types.MethodType(Ana.bla, a)
<bound method ?.bla of <__main__.Ana instance at 0x1093e1950>>
>>> types.MethodType(Ana.bla, a, Ana)
<bound method Ana.bla of <__main__.Ana instance at 0x1093e1950>>
>>> setattr(a, "__int__", types.MethodType(Ana.bla, a))
>>> int(a)
4
```